go/chat/boxer.go

```diff
@@ -219,6 +222,10 @@ func (b *Boxer) boxMessageV1(ctx context.Context, msg chat1.MessagePlaintextV1,
 		}
 	}
 
+	if len(msg.ClientHeader.TlfName) == 0 {
```

Comment:
We should never have RPC incompatibilities BTW.  Please make all RPC protocol changes backwards compatible